app/src/locale/messages.de.xlf

```diff
@@ -420,6 +420,21 @@
         <source>Photography: </source>
         <target state="final">Foto: </target>
       </trans-unit>
+      <trans-unit id="links" datatype="html">
+        <source>Links</source>
+        <target state="final">Links</target>
+        <note priority="1" from="description">Links label</note>
+      </trans-unit>
+      <trans-unit id="nationality" datatype="html">
```

Comment:
please remove `<trans-unit id="citizenship" datatype="html">` above for `nationality`, i think `citizenship` is no longer needed.